front/SettingsForm.tsx

```diff
@@ -99,13 +102,15 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="password"
               placeholder="New Password"
+              data-cy="password-settings"
               value={userInfo.password}
               onChange={updateState('password')}
               autoComplete="new-password"
             />
           </fieldset>
           <button
-            className="btn btn-lg btn-primary pull-xs-right"
+            className="btn btn-lg btn-primary pull-xs-right" 
+            data-cy="update-btn"
```

Comment:
Consider using a more descriptive name for the data-cy attribute, such as 'update-settings-btn'.